main.go

```diff
@@ -0,0 +1,197 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// A Recipe Struct allows you to insert recipe documents into your
+// collection
+
+type Recipe struct {
+	Name              string
+	Ingredients       []string
+	PrepTimeInMinutes int `json:"prepTimeInMinutes" bson:"prepTimeInMinutes"`
```

Comment:
For the PrepTimeInMinutes field, you include marshal/unmarshal info, but is that necessary? If so, wouldn't that apply to "Name" and "Ingredients" too? It could be useful to explain if PrepTimeInMinutes is an outlier.

Reply:
this is only necessary here so that atlas uses the key name in camel case. If it was ```PrepTimeInMinutes int``` atlas would make the key name "preptimeinminutes" instead of "prepTimeInMinutes"